### PR TITLE
Refactor/apollo client

### PR DESCRIPTION
**Pull request by topher-nullset**
## Description
should have done this sooner...
logic for which uri to hit is now automatic

## Related Issue
[Link to any related issue(s) that this PR addresses, e.g., "Closes #123"]

## Checklist
- [x] I have reviewed the code changes.
- [x] I have tested this code.
- [x] I have requested an update to the documentation.
- [x] I have checked for and resolved any merge conflicts.
- [x] I have assigned reviewers for this PR.

### CODE REVIEW

File: src/ApolloClient/client.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
 import {ApolloClient, InMemoryCache} from '@apollo/client';
-const client = new ApolloClient
-({
-  uri: 'https://fff-be-2e7913919a6b.herokuapp.com/graphql',
-  // uri: 'http://127.0.0.1:6969/graphql',
+
```
Comment (topher-nullset) [inline src/ApolloClient/client.js:2]: This line imports necessary modules from the Apollo Client library. ApolloClient is the main class representing the GraphQL client, and InMemoryCache is used for caching GraphQL query results in memory.

File: src/ApolloClient/client.js [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
+const uri = process.env.NODE_ENV === 'production'
+  ? 'https://fff-be-2e7913919a6b.herokuapp.com/graphql'
+  : 'http://127.0.0.1:6969/graphql';
+
```
Comment (topher-nullset) [inline src/ApolloClient/client.js:6]: This code block checks the current environment (NODE_ENV). If the application is running in production mode, it uses the production GraphQL API endpoint hosted on Heroku (https://fff-be-2e7913919a6b.herokuapp.com/graphql). If it's in development mode, it uses the local GraphQL API endpoint (http://127.0.0.1:6969/graphql).

File: src/ApolloClient/client.js [hunk 3]
```diff
@@ -1,8 +1,11 @@
 import {ApolloClient, InMemoryCache} from '@apollo/client';
-const client = new ApolloClient
-({
-  uri: 'https://fff-be-2e7913919a6b.herokuapp.com/graphql',
-  // uri: 'http://127.0.0.1:6969/graphql',
+
+const uri = process.env.NODE_ENV === 'production'
```
Comment (topher-nullset) [inline src/ApolloClient/client.js:3]: In JavaScript, const is a keyword used to declare a constant variable. Constants are block-scoped, meaning they are accessible only within the block of code where they are defined. Once a value is assigned to a constant using const, it cannot be reassigned. Here's an example:

**Review by BenHolden010 — APPROVED** [review]
looks professional! this is very dynamic and intuitive.

**Pull request merged**